datasets/lu/administrative_sanctions/crawler.py

```diff
@@ -0,0 +1,97 @@
+from typing import cast
+
+from zavod import Context, helpers as h
+from datetime import datetime
+
+FRENCH_TO_ENGLISH_MONTHS = {
+    "janvier": "january",
+    "février": "february",
+    "mars": "march",
+    "avril": "april",
+    "mai": "may",
+    "juin": "june",
+    "juillet": "july",
+    "août": "august",
+    "septembre": "september",
+    "octobre": "october",
+    "novembre": "november",
+    "décembre": "december",
+}
+
+
+def parse_date(date: str) -> str:
+    for french_month in FRENCH_TO_ENGLISH_MONTHS:
+        date = date.replace(french_month, FRENCH_TO_ENGLISH_MONTHS[french_month])
+
+    try:
+        return datetime.strptime(date, "%d %b %Y")
+    except:
+        return None
+
+
+def crawl_item(url: str, context: Context):
+    response = context.fetch_html(url)
+
+    # The title is in the format "Sanction administrative du XX XXXX 20XX"
+    title = response.find('.//*[@class="single-news__title"]')
+    date = title.text_content().strip().replace("Sanction administrative du ", "")
+    subtitle = (
+        response.find('.//*[@class="single-news__subtitle"]').text_content().strip()
+    )
+    res = context.lookup("subtitle_to_names", subtitle)
+    if res:
+        names = cast("List[str]", res.names)
+    else:
+        context.log.warning("Can't find the name of the company", text=subtitle)
+        names = [subtitle]
+
+    # If the subtitle doesn't contain any names
+    if not names or len(names) == 0:
+        return
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(*names)
+    for name in names:
+        entity.add("name", name)
+
+    entity.add("topics", "reg.warn")
```

Comment:
I think it in indeed makes more sense to use `crime.fin`, I used `reg.warn` because it was what @pudo suggested [here](https://github.com/opensanctions/crawler-planning/issues/157#issuecomment-2102134168)

Reply:
oh I didn't see that - no I think crime.fin is fine then